requirements/requirements.in

```diff
@@ -8,3 +8,4 @@ djangorestframework
 django-crum
 django-split-settings
 inflection
+sqlparse==0.5.0 # https://github.com/ansible/django-ansible-base/security/dependabot/9
```

Comment:
The link gives `sqlparse>=0.5.0` to get a patched version, and in the `.in` files, we really want to be as unoppionated as possible, because it's almost inevitable to have conflicts with downstream requirements.

Reply:
Agree on this. We need to avoid creating a situation where every dependency bump will require a new gateway release.

Reply:
I must be missing something here, why are we worried about downstream requirements? Aren't all the requirements starting from here? And why is having new DAB release such a problem?
Eventually we will need to pin the dependencies somewhere.